datasets/huishoudelijkafval/ticket/v1.0.0.json

```diff
@@ -0,0 +1,249 @@
+                "description": "Identificerend kenmerk van de containerlocatie."
+            },
+            "datumCreatie": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket aangemaakt is."
+            },
+            "plaatje": {
+                "type": "array",
+                "provenance": "plaatje_url",
+                "description": "De url van de bijgevoegde foto."
+            },
+            "datumFactuur": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket gefactureerd is."
+            },
+            "factuurNummer": {
+                "type": "string",
+                "description": "Het nummer van de factuur."
+            },
+            "datumWijziging": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket is gewijzigd."
+            },
+            "referentienummerLeverancier": {
+                "type": "string",
+                "provenance": "referentienummer_vconsyst_service",
+                "description": "Het referentienummer dat door de leverancier aan het ticket gegeven wordt."
+            },
+            "prioriteitOpmerking": {
+                "type": "string",
+                "description": "Een opmerking die toegevoegd wordt om de prioritering te verduidelijken."
+            },
+            "prioriteitNaam": {
+                "type": "string",
+                "description": "De naam die de soort prioriteit weergeeft."
+            },
+            "prioriteitResponstijd": {
+                "type": "integer",
+                "description": "De toegestane responstijd voor deze prioriteit. "
+            },
+            "probleemmoduleNaam": {
+                "type": "array",
+                "description": "De naam van de probleemmodule."
+            },
+            "onderhoudsbedrijf": {
+                "type": "string",
+                "provenance": "naam_onderhoudsbedrijf",
+                "description": "De naam van het onderhoudsbedrijf."
+            },
+            "eigenaar": {
+                "type": "string",
+                "provenance": "naam_eigenaar",
+                "description": "De naam van de eigenaar van de container."
+            },
+            "status": {
+                "type": "string",
+                "provenance": "status_ticket",
+                "description": "De status van het ticket."
+            },
+            "naamTickettype": {
+                "type": "string",
+                "provenance": "ticket_type_name",
+                "description": "De naam van het type ticket."
+            },
+            "rangordeNummerTickettype": {
+                "type": "integer",
+                "provenance": "ticket_type_rangorde_nummer",
+                "description": "Het rangordenummer van het type ticket."
+            },
+            "datumWorkflowStart": {
+                "type": "datetime",
```

Comment:
From here 10 more `datetime` fields that need to be corrected.

Reply:
alles is aangepast